cmd/ddev/cmd/commands.go

```diff
@@ -108,7 +108,7 @@ func addCustomCommands(rootCmd *cobra.Command) error {
 					usage = val
 				}
 				if val, ok := directives["Example"]; ok {
-					example = val
+					example = "  " + strings.ReplaceAll(val, `\n`, "\n  ")
```

Comment:
I wish there was an easier way to do this (for the command writer) but I'm fine with this. It's far easier to parse things as lines than otherwise.

Reply:
Yeah same thought. There are ways but I guess this is the easiest solution also for the users.